packages/gatsby/src/redux/reducers/last-action.ts

```diff
@@ -1,6 +1,6 @@
 import { IGatsbyState, ActionsUnion } from "../types"
 
-export const lastAction = (
-  _state: IGatsbyState["lastAction"],
+export const lastActionReducer = (
+  _state: unknown,
```

Comment:
Why change to `unknown` here?

Reply:
As the `_state` parameter is never used in this reducer, if I try to keep the current type `IGatsbyState["lastAction"]` here, the typing validation (typecheck) generates some errors. Using the `unknown` type, the typecheck performed without errors and, apparently, with no side effects.

Reply:
I tried to use the `never` type as well but the type error kept happening.